tests/mock/mockProfile/profiles.yml

```diff
@@ -1,5 +1,24 @@
 fal_test:
   target: dev
+  fal_extract_load:
+    dev:
+      - name: 'my_fivetran_el'
+        type: fivetran
+        api_key: my_fivetran_key
+        api_secret: my_fivetran_secret
+        connectors:
+          - name: fivetran_connector_1
+            id: id_fivetran_connector_1
+          - name: fivetran_connector_2
+            id: id_fivetran_connector_2
+      - name: 'my_airbyte_el'
+        type: airbyte
+        host: http://localhost:8001
+        connections:
+          - name: airbyte_connection_1
+            id: id_airbyte_connection_1
+          - name: airbyte_connection_2
+            id: id_airbyte_connection_2
```

Comment:
What we had proposed looked a bit different, with `my_fivetran_el` being a document key and the rest of the information being a document:

```yaml
fal_extract_load:
  dev:
    my_fivetran_el:
      type: fivetran
# ...
```
This is good because it assures us there is no repeated `name` value (and that if there is, yaml catches that)